src/jit.rs

```diff
@@ -383,9 +413,7 @@ impl<'a> FunctionTranslator<'a> {
         let local_id = self
             .module
             .declare_data_in_func(sym, &mut self.builder.func);
-
-        let pointer = self.module.target_config().pointer_type();
-        self.builder.ins().symbol_value(pointer, local_id)
+        self.builder.ins().symbol_value(self.int, local_id)
```

Comment:
`self.int` is not a valid type for a pointer.

Reply:
```rust
// Our toy language currently only supports I64 values, though Cranelift
        // supports other types.
        let int = self.module.target_config().pointer_type();
```

```rust
// Now translate the statements of the function body.
        let mut trans = FunctionTranslator {
            int,
            builder,
            variables,
            module: &mut self.module,
            data_ctx: &mut self.data_ctx,
        };
```

`self.int` should be the pointer type here?

Reply:
I see. I expected it to be the size of a c int, which is generally 32bit.

Reply:
Can we call the field something else? Maybe `intptr` or something?